public/app/features/templating/formatRegistry.ts

```diff
@@ -254,6 +255,14 @@ export const formatRegistry = new Registry<FormatRegistryItem>(() => {
         return formatQueryParameter(name, value);
       },
     },
+    {
+      id: FormatRegistryID.jsonWithoutQuote,
+      name: 'JSON',
+      description: 'JSON stringify valu',
```

Comment:
We should change the text here to explain what it is for.

Reply:
It has been modified,